docs/src/P3Scheme.md

```diff
@@ -133,18 +133,29 @@ As a result ``q\_{ice}`` can be expressed as a sum of inclomplete gamma function
 where ``\Gamma \,(a, z) = \int_{z}^{\infty} \! t^{a - 1} e^{-t} \mathrm{d}D``
   and ``\Gamma \,(a) = \Gamma \,(a, 0)`` for simplicity.
 
+Within our solver, we approximate ``\mu`` from q/N and keep it constant throughout the solving step. We approximate ``\mu`` by an exponential function given by the q/N points corresponding to ``\mu = 6`` and ``\mu = 0``. This is shown below as well as how this affects the solvers ``\lambda`` solutions.
```

Comment:
Is it exponential or linear approximation?

Reply:
It is linear on a log scale of q/N so it is exponential but appears linear on our plots. I did also try a purely linear approximation and I believe it is slightly worse (though still works)